apps/govern/components/Layout/Balance.tsx

```diff
@@ -0,0 +1,44 @@
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Button, Tooltip, Typography } from 'antd';
+import { useVotingPower } from 'hooks/index';
+import { useAccount } from 'wagmi';
+
+import { COLOR } from 'libs/ui-theme/src/lib/ui-theme';
+
+import { UNICODE_SYMBOLS } from 'common-util/constants/unicode';
+import { GET_VEOLAS_URL } from 'common-util/constants/urls';
+import { formatWeiBalance } from 'common-util/functions';
+
+const { Text, Paragraph } = Typography;
+
+const Balance = () => {
+  const { address: account } = useAccount();
+  const { data, isFetching } = useVotingPower(account);
+
+  if (isFetching) return null;
+  if (data === undefined) return null;
+
+  return (
+    <Tooltip
+      color={COLOR.WHITE}
+      title={
+        <>
+          <Paragraph>veOLAS gives you voting power in Autonolas governance.</Paragraph>
+          <a href={GET_VEOLAS_URL} target="_blank">
+            Lock OLAS for veOLAS {UNICODE_SYMBOLS.EXTERNAL_LINK}
+          </a>
+        </>
+      }
+    >
+      <Button type="text">
```

Comment:
strange to use a Button here if it doesn't do anything

Reply:
it's used as a tooltip trigger. and should have hover state. do you think it's better to use just a div?

Reply:
ah i see, then sure is fine imo